packer/netbsd_openbsd.pkr.hcl

```diff
@@ -97,11 +49,25 @@ source "qemu" "qemu-gce-builder" {
 }
 
 build {
-  name="netbsd-vanilla"
-  sources = ["source.qemu.qemu-gce-builder"]
+  name = "vanilla"
+
+  dynamic "source" {
+    for_each = var.vanilla_name
+    labels = ["source.qemu.qemu-gce-builder"]
+    iterator = tag
+
+    content {
+      name = tag.value.name
+    }
+  }
```

Comment:
What is this for?

Reply:
This is mainly for using `-only "*.${CIRRUS_TASK_NAME}"` while building net/openbsd images, so we can easily combine the packer invocations later. Like you mentioned in .cirrus.yml, we can't use `var or local for` build.name so I implemented it in this way.